### PR TITLE
docs: Improve visibility for `.enabled()` etc., add example

### PR DESCRIPTION
**Pull request by nh2**
Following discussion with @xclaesse on IRC.

### CODE REVIEW

File: docs/markdown/Reference-manual.md [hunk 1]
```diff
@@ -2000,6 +2000,16 @@ Dictionaries are available since 0.47.0.
 Since 0.48.0 dictionaries can be added (e.g. `d1 = d2 + d3` and `d1 += d2`).
 Values from the second dictionary overrides values from the first.
 
+### Feature option object
```
Comment (xclaesse) [inline docs/markdown/Reference-manual.md]: This should be moved in the `## Returned objects` section just below.

Comment (jon-turney) [reply docs/markdown/Reference-manual.md]: also, there should be a link to this from the text in `get_option()` above which mentions these objects.

Comment (nh2) [reply docs/markdown/Reference-manual.md]: Both done.

**Comment by nh2** [comment]
Feedback addressed, please have another look.

File: docs/markdown/Build-options.md [hunk 2]
```diff
@@ -94,6 +94,21 @@ if d.found()
 endif
 ```
 
+To check the value of the feature, the object has three methods
+returning boolean and taking no argument:
```
Comment (jon-turney) [inline docs/markdown/Build-options.md]: "returning a boolean"

Comment (nh2) [reply docs/markdown/Build-options.md]: Done.

**Pull request merged**